CONTRIBUTING.md

```diff
@@ -43,6 +44,16 @@ export TEST_ACCOUNT_ADDRESS=0x065A822f0000000000000000000000000c26641
 export TEST_ACCOUNT_PRIVATE_KEY=0x02a80000000000000000000000001754438a
 ```
 
+The global variables above will only be valid for some of the tests.
+The recommended and more straightforward approach is to go with the docker.
+You just need to do the following steps:
+
+- Install the [docker](https://docs.docker.com/desktop/install/mac-install/) (You can install it via `brew` or some other package manager)
+- Run the `docker` on your machine (Open the application).
```

Comment:
```suggestion
- Install [Docker](https://docs.docker.com/engine/install/) (it can also be installed via a package manager, e.g. `brew` for Mac)
- Run `Docker` on your machine (open the application).
```